packages/sanity/src/core/bundles/util/util.ts

```diff
@@ -1,17 +1,24 @@
 import speakingurl from 'speakingurl'
 
 import {type BundleDocument} from '../../store/bundles/types'
+import {isVersionId} from '../../util'
 
 const PUBLISHED_SLUG = 'Published'
+const DRAFTS_SLUG = 'Drafts'
 
 /**
  * @internal
  * @hidden
  */
 export function getBundleSlug(documentId: string): string {
```

Comment:
I think we'd rather want this to return `undefined` if we can't determine a bundle identifier, but we can address this in a follow-up pr

Reply:
Done here: https://github.com/sanity-io/sanity/pull/7357/commits/a362dee82bb47cef9ad3404d75000dd79759d107